packages/core/src/Asset.ts

```diff
@@ -1,6 +1,6 @@
 import { extractTagsFromFileName } from './utils/extractTagsFromFileName';
 import { getHash } from './utils/getHash';
-import fs from 'fs-extra';
+import { readFileSync } from 'fs-extra';
```

Comment:
we can't import like this because of commonjs and es module compatibility

can you revert this change